icevision/models/torchvision/lightning_model_adapter.py

```diff
@@ -36,17 +36,16 @@ def training_step(self, batch, batch_idx):
 
     def validation_step(self, batch, batch_idx):
         (xb, yb), records = batch
-        with torch.no_grad():
-            self.train()
-            train_preds = self(xb, yb)
-            loss = loss_fn(train_preds, yb)
-
-            self.eval()
-            raw_preds = self(xb)
-            preds = self.convert_raw_predictions(
-                batch=batch, raw_preds=raw_preds, records=records
-            )
-            self.accumulate_metrics(preds=preds)
+        self.train()
+        train_preds = self(xb, yb)
+        loss = loss_fn(train_preds, yb)
```

Comment:
I wasn't able to understand why do we calculate training prediction here?

Reply:
Maybe the name is misleading , these are in fact validation preds. These are used to calculate validation loss and metrics